### ISSUE TITLE
MVD Container Deployment

### ISSUE THREAD

**Issue by zeier** [agera-edc/MinimumViableDataspace#183]
After #157

## Description

#157 provide a means to run the MVD on a local machine without any dependencies to cloud services, which is a setup that is useful for hackathons and development purposes.

Many parties intend to deploy their dataspaces using container orchestrators like AKS or OpenShift. Providing the corresponding Helm charts to make deployments easier will be useful for many use cases.

This feature should however not change the current deployment approach that uses ACI, because introduction of a container orchestrator will add more complexity to the MVD.

## Acceptance Criteria

- [ ] Helm charts are available for MVD components
- [ ] MVD can be deployed on a local machine using Minikube
- [ ] A local file transfer between two participants can be performed

## Out of Scope

- [ ] Deployment pipeline (GitHub Actions) to deploy the MVD to e.g. AKS

## Stories

- [ ] Local MVD Deployment with Helm Charts

**Comment by zeier** [comment]
This item is now covered by upstream issue https://github.com/eclipse-dataspaceconnector/MinimumViableDataspace/issues/23
